src/pages/roadmap/index.tsx

```diff
@@ -0,0 +1,176 @@
+    items: [
+      {
+        name: "Design tools integrations",
+        description: `With Tolgee, developers and translators can 
+        translate the apps very easily. We would like to provide the simplicity
+        also to translators by enabling them to translate texts directly in 
+        tools like <b>Figma, XD or Miro</b>`,
+      },
+      {
+        name: "Support for mobile apps",
+        description: `Tolgee works great with Web apps and we would like to provide the
+          same comfort to mobile developers.`,
+      },
+      {
+        name: "QA Checks",
+        description: `Even translators make mistakes, but some of these mistakes
+         can be detected and translators can be notified. No more mistakes like 
+         missing tags, missing parameters or missing punctuation marks`,
+      },
+      {
+        name: "Git integrations",
+        description: [
+          `Sometimes it would be great if localization data are 
+           synced with your git repository. Or maybe you would like to create pull
+           request automatically when translations are changed....`,
+          `To do this, integrations to <b>GitHub, GitLab</b> or just
+           support for Git itself could be helpful`,
+        ],
+      },
+      {
+        name: "Branching",
+        description: `Sometimes, you need to maintain multiple versions of your software,
+           so you would like to have multiple versions of your localization 
+           data stored in Tolgee as well.`,
+      },
+      {
+        name: "Translator Marketplace",
+        description: `Looking for translator to access foreign markets? Tolgee will help you with this. 
+          With Tolgee Translator Marketplace, there will be nothing easier 
+          than finding the ideal translator for your app.`,
+      },
+    ],
+  },
+  "2021/Q4": {
+    active: true,
+    items: [
+      {
+        name: "Glossaries",
+        description: `To translate the same terms same way every time,
+           you need to store the terminology to reuse it in the future.
+           Glossaries are the feature enabling you to do that.`,
+      },
+      {
+        name: "Translation memory",
+        description: `Translation memory will suggest you sentences, which
+         were already translated in your projects before.`,
+      },
+
+      {
+        name: "Automated translations",
+        description: `Don't translate everything manually. With automated
+         translations feature, services like DeepL, Google Translate
+          or AWS translate will help you to translate your strings automatically.`,
+      },
+      {
+        name: "Paywall for Tolgee Cloud",
+        description: `To generate some profit from providing this great localization
+         tool (Yes, we mean Tolgee), we need to enable our users to pay for our 
+         services. To do so, we have to implement backend services and UI to bill our services.`,
+      },
+    ],
+  },
+  "2021/Q3": {
```

Comment:
What we have done so far: (looks confusing without this explanation as it is not chronological)